docs/reference/api/pages.md

```diff
@@ -373,6 +373,48 @@ The successful PATCH request returns the complete patched document. See the [pag
 
 The trash is part of the overall page tree in order to maintain the nesting structure. As such, there is not only a simple `trash` property to set `true`. Instead, set `_targetId` to `_trash` and `_position` to `lastChild` (or another position within the trash). You may similarly move pages out of the trash by moving them to a position relative to another page that is not in the trash.
 
+## `DELETE /api/v1/@apostrophecms/page/:_id`
+
+
+**Authentication required.**
+
+This API route **permanently deletes the page database document**. Moving pieces to the trash in the Apostrophe user interface or [using a PATCH request](#moving-pages-to-the-trash) do not permanently delete database documents and are typically better options.
+
+DELETE requests will be rejected if:
+- the `_id` matches the draft mode of a page that has an existing published mode document, or
```

Comment:
Maybe we don't need 'or' if this is a list. Unless there is an 'and' hidden somewhere